source/_docs/automation/trigger.markdown

```diff
@@ -291,6 +291,16 @@ automation:
 
 The `for` template(s) will be evaluated when an entity changes as specified.
 
+<div class='note warning'>
```

Comment:
Its not just this but many other things.

I'm not sure if this needs such an extensive note. It almost looks like you want to say: Don't use `for`. But that would not be our projects recommendation at all.

Reply:
It's not just a blanket "don't use it" but instead a warning to not use it for critical automation triggers that need to survive restarts/re-loads. I don't see a problem using it as long as the user knows its limitations and can code resiliency into their automations.

I'm not sure how to make it more concise and still convey the message that using the option is not as reliable as one would think from the current documentation. Right now there is no inking at all in the documentation that the time trigger will fail under certain circumstances.

However, I actually am typically recommending that people not use any time based functions (including for:) that are longer than just a few minutes in automations or scripts when the topic gets brought up in the forums for this reason.

And as for the "many other things" this is just the first documentation change to get a feel for what will be accepted before adding warnings to all the other time based functions to give appropriately similar warnings.

Reply:
Still think this warning needs to be limited more.

Reply:
feel free to make a suggestion that limits it but still correctly describes the concern and offers a better alternative.

I can't see a way to do that.